pkg/proxy/proxy.go

```diff
@@ -63,6 +66,8 @@ type Config struct {
 
 	// Force use of unencrypted ws:// protocol instead of wss://
 	NoWSS bool
+
+	supportedEvents map[string]bool
```

Comment:
I think this should be a property of `Proxy`, not `Config`. Maybe we should have a `NewProxy` function to initialize the `Proxy` struct and do the conversion from slice to map.